ee-9/feature-pack/pom.xml

```diff
@@ -54,6 +54,7 @@
         <version.jakarta.authentication.jakarta-authentication-api>2.0.0</version.jakarta.authentication.jakarta-authentication-api>
         <version.jakarta.authorization.jakarta-authorization-api>2.0.0</version.jakarta.authorization.jakarta-authorization-api>
         <version.jakarta.batch.jakarta.batch-api>2.0.0</version.jakarta.batch.jakarta.batch-api>
+        <version.org.jberet>2.0.1.Final</version.org.jberet>
```

Comment:
@parsharma This property is already declared in line 93 of this file. Please update that one.